Formula/offlineimap.rb

```diff
@@ -1,10 +1,11 @@
 class Offlineimap < Formula
   desc "Synchronizes emails between two repositories"
   homepage "https://www.offlineimap.org/"
-  url "https://files.pythonhosted.org/packages/09/12/73db8d38fea8ec3536cbccb8286b46b426639aff7e166840fa5e68e889e2/offlineimap-7.3.4.tar.gz"
-  sha256 "5dbd7167b8729d87caa50bed63562868b6634b888348d9bc088a721530c82fef"
+  url "https://github.com/OfflineIMAP/offlineimap3/archive/v8.0.0.tar.gz"
```

Comment:
Are they no longer publishing versions at PyPI?

Reply:
Actually, it seems like this should be a separate formula, no? It's a different repository from `offlineimap`.

Reply:
@carlocab I followed the same strategy archlinux followed. They simply pumped the offlineimap base python installation from python 2 to python 3.
https://github.com/archlinux/svntogit-community/commits/packages/offlineimap/trunk/PKGBUILD

Reply:
offlineimap3 is not published on PYPI no

Reply:
@carlocab I can put it as a separate Formula if you like. Not my place to choose :).
Reason why I did it this way: I want Python2 gone as it is no longer maintained.

Reply:
How different is `offlineimap` from `offlineimap3`, other than the Python version used?

Reply:
Should be an in-place replacement. Rewrite in Python3 that is all.